MODULE.bazel

```diff
@@ -4,9 +4,12 @@ module(
     compatibility_level = 1,
 )
 
-# Note: rules_license must not depend on any other repositories if you are
-# just using basic rules under //rules/... and //licenses.
+# NOTE: rules_license must not depend on any other repositories if you are
+# just using basic rules under //rules/... and //licenses/...
 
 # TODO(aiuto): Create an extension to enable the rules under //tools/...
 # That will require rules_python, which we do not want to force on people who
 # do not need //tools.
+
+# Only for development
+bazel_dep(name = "rules_pkg", version = "0.7.0", dev_dependency = True)
```

Comment:
Hi Tony, it is my interpretation from reading this diff that you are retrieving the `rules_pkg` dep from the BCR instead of with `http_archive` from a github mirror. This change looks good to me if that is your intent.

Are there any compatibility issues for a bazel project that is not using a bazel version that supports bzlmod?

Reply:
That is the intent.
Projects not using bzlmod are unchanged, because they pick up rules_pkg from the WORKSPACE file.